### PR TITLE
spectrwm: update to 3.5.1, orphan

### PR DESCRIPTION
**Pull request by tmtrucco**
#### Testing the changes
I tested the changes in this PR: YES

#### Local build testing
I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/spectrwm/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'spectrwm'
 pkgname=spectrwm
-version=3.4.1
-revision=3
+version=3.5.1
+revision=4
```
Comment (mhmdanas) [inline srcpkgs/spectrwm/template:4]: `revision` should be reset to 1 on version bumps.

File: srcpkgs/spectrwm/template [hunk 2]
```diff
@@ -10,11 +10,11 @@ hostmakedepends="pkg-config"
 makedepends="libXcursor-devel libXft-devel libXrandr-devel libXt-devel
  xcb-util-keysyms-devel xcb-util-wm-devel"
 short_desc="Small dynamic tiling window manager for X11"
-maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+maintainer=" Orphaned <orphan@voidlinux.org>"
```
Comment (mhmdanas) [inline srcpkgs/spectrwm/template:13]: Remove the extra space

**Review by mhmdanas — CHANGES_REQUESTED** [review]
Thanks for your PR! Please don't keep closing your PRs and opening new ones just to make changes to them; instead, amend your commits and force-push to the existing PR branch (see https://www.atlassian.com/git/tutorials/rewriting-history).

**Comment by tmtrucco** [comment]
Sorry for the PR spam! I made the first PR with just the web and couldn't
figure out how to ammend the commit. I started all over again from terminal
with git so I can finally push the ammend. It should be good to go now,
sorry for the inconveniences

**Comment by mhmdanas** [comment]
No worries, looks good to me now.

**Pull request merged**